README.md

```diff
@@ -192,13 +203,22 @@ users = users_resp["users"]
 
 You can manage SSO settings and map SSO group roles and user attributes.
 
+Note: When sending certificates, send only the contents without the start prefix
+and and suffix
+
+```
+-----BEGIN CERTIFICATE-----
```

Comment:
I actually meant putting this value in:
```
idp_cert="1231232132131",
```

doesn't the api/sdk support both formats? with and without prefox+suffix ?